src/profile-logic/errors.js

```diff
@@ -10,14 +10,16 @@ import type { RequestedLib } from '../types/actions';
 // symbols for a specific library
 export class SymbolsNotFoundError extends Error {
   library: RequestedLib;
-  error: ?Error;
+  errors: Error[];
 
-  constructor(message: string, library: RequestedLib, error?: Error) {
-    super(message);
+  constructor(message: string, library: RequestedLib, errors: Error[] = []) {
```

Comment:
Optional: This could be `constructor(message: string, library: RequestedLib, ...errors: Error[])` for a slightly more ergonomic constructor.

Reply:
Good idea!